src/sentry/snuba/outcomes.py

```diff
@@ -203,8 +203,8 @@ def get_filter(
 
     if "project_id" in params:
         filter_keys["project_id"] = params["project_id"]
-    if "organization" in params:
-        filter_keys["organization_id"] = params["organization_id"]
+    if "organization_id" in params:
```

Comment:
this was off, but never noticed since we have some logic that automatically infers the org id based on projects passed in

Reply:
Should we add a test to check that the filter keys generated here are correct?

Reply:
@wmak the endpoints tests were failing without this since now project_ids can be empty and there's no org to infer from them, but will add a couple quick unit tests to QueryDefinition